docs/userguide.html

```diff
@@ -83,8 +83,7 @@ <h4 id="wildfire-susceptibility">Wildfire Susceptibility</h4>
 
 <h3 id="drought">
 <a class="anchor" href="#drought" aria-hidden="true"><span class="octicon octicon-link"></span></a>Drought</h3>
-<p>To estimate an asset’s exposure to drought, we utilized a drought index based on CMIP6 simulations of precipitation and temperature, from which we estimate potential evapotranspiration. This is the more suitable index for agricultural assets like farms and farm production sites. The drought index, also referred to as the <a href="https://spei.csic.es/home.html">standardized precipitation evapotranspiration index (SPEI)</a>, represents the magnitude of precipitation deficits (negative magnitude) or surplus (positive magnitude) along with accounting for evaporation over the preceding 12-month period. Values between -1.5 and -2.0 indicate moderate dryness, while values below -2.0 indicate severe dryness. High risk, exceptional droughts fall below -2.0 on the standard precipitation evapotranspiration index (SPEI). Drought index values are calculated relative to a historical 1950-2014 baseline period, and future droughts can reach index values below -3.0 if the droughts are unprecedented relative to that 1950-2014 historical period.</p>
-
+<p>To estimate an asset’s exposure to drought, we utilized a drought index based on CMIP6 simulations of precipitation and temperature, from which we estimate potential evapotranspiration. This is the more suitable index for agricultural assets like farms and farm production sites. The drought index, also referred to as the standardized precipitation evapotranspiration index [SPEI] (https://spei.csic.es/home.html), represents the magnitude of precipitation deficits (negative magnitude) or surplus (positive magnitude) along with accounting for evaporation over the preceding 12-month period. Values between -1.5 and -2.0 indicate moderate dryness, while values below -2.0 indicate severe dryness. High risk, exceptional droughts fall below -2.0 on the standard precipitation evapotranspiration index (SPEI). Drought index values are calculated relative to a historical 1950-2014 baseline period, and future droughts can reach index values below -3.0 if the droughts are unprecedented relative to that 1950-2014 historical period.</p>
```

Comment:
@jasSust you can see that the SPEI link is no longer rendering properly in HTML. Please revert the corresponding change on line 36 in userguide.md

Reply:
@bcwaldon Ahh, got it -- should be fixed, now!